app/src/components/testCaseList/pagination/pagination.jsx

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useMemo, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { useIntl } from 'react-intl';
+import { InputDropdownSorting } from 'components/inputs/inputDropdownSorting';
+import {
+  PAGE_INCREMENT,
+  DECIMAL_RADIX,
+  PERCENTAGE_MULTIPLIER,
+  FIRST_PAGE,
+} from './pagination.constants';
+import { messages } from './messages';
+import styles from './pagination.scss';
+
+const cx = classNames.bind(styles);
+
+export const Pagination = ({
```

Comment:
there's already ui-kit component for that, is there a reason why you did not use it? 

https://reportportal.io/ui-kit/?path=/docs/navigation-pagination--docs